spec/services/course_exports/prepare_students_export_service_spec.rb

```diff
@@ -44,15 +44,15 @@
 
   let!(:student_5) { create :student, cohort: cohort_2_live, user: user_5 }
 
-  let(:target_group_l1_non_milestone) do
+  let(:target_group_l1_2) do
     create :target_group, level: level_1, sort_index: 0
   end
 
-  let(:target_group_l1_milestone) do
+  let(:target_group_l1_1) do
     create :target_group, level: level_1, sort_index: 1
   end
```

Comment:
It irks me that it's `target_group_l1_2` instead of `target_group_l1_0`. The `sort_index` is `0`, isn't it? 😅 

There's another `target_group_l2_1` below this that has `sort_index` `0`. So... what's going on with the numbering?

Reply:
@harigopal I didn't name based on `sort_index`, I just named them `1` and `2`. Should I change it ?

Reply:
Yes, it irks 🙂

Fixed it https://github.com/pupilfirst/pupilfirst/pull/1622/commits/b3593bc106e6ce411bda8c39efd828f2e8599b69